[RU] Add `osu! wiki` (#7838)
ppy/osu-wiki: wiki/osu!_wiki/ru.md

## Структура

Входом в osu! wiki является [главная страница](/wiki/Main_Page), разделённая на несколько высокоуровневых категорий, каждая из которых имеет общее описание и содержит список важных статей. Статьи, не перечисленные на главной странице, как правило, являются более подробными и могут быть доступны в виде гиперссылок на других страницах, а также через функцию поиска на сайте.

Отдельная статья описывает один термин или понятие и дает общий взгляд на связанные с ним темы. Чтобы лучше раскрыть тему, статья разбивается на подразделы, охватывающие различные аспекты термина. Статья также может включать необходимые скриншоты или другие визуальные пояснения, или ссылки на другие статьи по теме.<!-- TODO: via the use of infoboxes. uncomment this when they are ready (https://github.com/ppy/osu-wiki/issues/5440) -->

Некоторые статьи доступны на других языках. Перевод статей осуществляется самими членами сообщества, исходя из [списка поддерживаемых сайтом языков](/wiki/Article_styling_criteria/Formatting#locales).

## Модель

> osu! wiki построена на принципе доступности: правки и переводы поощряются и могут осуществляться любым способным и знающим человеком. Данные для статей хранятся в [репозитории GitHub](https://github.com/ppy/osu-wiki) и обновляются участниками путём [отправки пулл-реквестов](/wiki/osu!_wiki/Contribution_guide), которые затем проверяются и объединяются [сопровождающими вики](/wiki/osu!_wiki/Maintenance/List_of_maintainers).

Reviewer: ```suggestion
osu! wiki построена на принципе доступности: правки и переводы поощряются и могут осуществляться любым способным и знающим человеком. Данные для статей хранятся в [репозитории GitHub](https://github.com/ppy/osu-wiki) и обновляются участниками путём [создания пулл-реквестов](/wiki/osu!_wiki/Contribution_guide), которые затем проверяются и отправляются в релиз [администраторами вики](/wiki/osu!_wiki/Maintenance/List_of_maintainers).
```

> Статьи оформляются с использованием формата [GitHub-flavoured Markdown](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/). Кроме того, движок сайта поддерживает необходимые для вики пользовательские функции, такие как перенаправления, оформление первых статей или улучшения форматирования.<!-- TODO: article on custom features of the wiki? it could help us ease the ASC a bit as well -->

Reviewer 2: - «пользовательские функции» означало бы, что редакторы могут самостоятельно что-то реализовывать, что есть неправда
- front matter здесь — это служебная информация: https://osu.ppy.sh/wiki/ru/Article_styling_criteria/Formatting#%D0%BC%D0%B5%D1%82%D0%B0%D0%B4%D0%B0%D0%BD%D0%BD%D1%8B%D0%B5
```suggestion
Для оформления статей используется [GitHub-flavoured Markdown](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/). Кроме того, в движке веб-сайта реализована дополнительные функции, критичные для вики: перенаправления, метаданные статей, дополнительные возможности разметки.
```

> Статьи составляются в соответствии с [критериями оформления статей](/wiki/Article_styling_criteria) osu! wiki, они написаны нейтральным языком и следуют общим правилам форматирования и оформления. Некоторые информационные статьи служат списком часто задаваемых вопросов (FAQ) по теме и используют неформальный язык. Переводы следуют принципу соответствия содержания и соблюдают одинаковые правила форматирования.

Reviewer 2: ```suggestion
Статьи составлены в соответствии с [критериями оформления](/wiki/Article_styling_criteria) osu! wiki: они написаны нейтральным языком и следуют общим правилам форматирования и оформления. Некоторые справочные статьи являются списками часто задаваемых вопросов (FAQ) и используют неформальный тон. Переводы выполняются по принципу равенства содержания и следуют общим правилам форматирования.
```

> Несмотря на наличие суффикса *wiki* в названии, osu! wiki не полностью удовлетворяет [концепции Wiki](https://en.wikipedia.org/wiki/Wiki):

Reviewer 2: ```suggestion
Несмотря на наличие суффикса *wiki* в названии, osu! wiki не полностью удовлетворяет [концепции Wiki](https://ru.wikipedia.org/wiki/Вики):
```

> - Улучшение существующих статей и создание новых поощряется, но для внесения вклада необходимы базовые знания [GitHub](https://github.com), что повышает порог вхождения. Однако существует специальное [руководство по работе с osu! wiki](/wiki/osu!_wiki/Contribution_guide), помогающее новым участникам легко приступить к работе.

Reviewer 2: мне кажется, так лучше. если нет, резкое начало предложения с «Однако» точно нужно переделать
```suggestion
- Улучшение существующих статей и создание новых поощряется, но для этого необходимы базовые знания [GitHub](https://github.com), что повышает порог вхождения. Чтобы новым участникам было легче приступить к работе, на вики есть отдельное [руководство по работе с ней](wiki/osu!_wiki/Contribution_guide).
```

> - Все изменения в osu! wiki предварительно модерируются и подвергаются рецензированию, в ходе которого участник должен сотрудничать с рецензентами и отвечать на их отзывы.

Reviewer 2: ```suggestion
- Все изменения в osu! wiki проходят премодерацию и ревью, в ходе которого автор правок должен взаимодействовать с ревьюерами и исправлять их замечания.
```

## Лицензия

> Большинство материалов в osu! wiki распространяются на условиях лицензии [CC-BY-NC 4.0](https://github.com/ppy/osu-wiki/blob/master/LICENCE.md). Это означает, что использование материалов вики разрешено в некоммерческих целях (см. [краткое содержание](https://tldrlegal.com/license/creative-commons-attribution-noncommercial-4.0-international-(cc-by-nc-4.0)])). Некоторые материалы могут быть защищены различными сторонними лицензиями, в этом случае они будут указаны в  локальной лицензии и будут иметь приоритет над глобальной лицензией.

Reviewer 2: ```suggestion
Большинство материалов в osu! wiki распространяются на условиях лицензии [CC-BY-NC 4.0](https://github.com/ppy/osu-wiki/blob/master/LICENCE.md). Это означает, что использование материалов вики разрешено в некоммерческих целях (см. [краткое содержание лицензии](https://tldrlegal.com/license/creative-commons-attribution-noncommercial-4.0-international-(cc-by-nc-4.0)])). Некоторые материалы могут быть защищены различными сторонними лицензиями, которые в таких случаях находятся рядом с материалом и имеют приоритет над глобальной лицензией.
```

> Обратите внимание, что это не распространяется на использование брендов "osu!" или "ppy" в любом программном обеспечении, ресурсах, рекламе или промоушене, поскольку они защищены законом о товарных знаках. Если вам требуется разрешение на использование этих терминов, пожалуйста, [свяжитесь с нами](mailto:contact@ppy.sh).

Reviewer 2: ```suggestion
Обратите внимание, что это не распространяется на использование брендов "osu!" или "ppy" в любом программном обеспечении, ресурсах, рекламе или продвижении, поскольку они защищены законом о товарных знаках. Если вам требуется разрешение на использование этих терминов, пожалуйста, [свяжитесь с нами](mailto:contact@ppy.sh).
```
